src/views/Farms/components/FarmCard/ApyButton.tsx

```diff
@@ -0,0 +1,45 @@
+import React from 'react'
+import BigNumber from 'bignumber.js'
+import { CalculateIcon, IconButton, useModal } from '@pancakeswap-libs/uikit'
+import ApyCalculatorModal from './ApyCalculatorModal'
+
+export interface TokenAddressesObject {
+  56?: string
+  97?: string
+}
```

Comment:
We should be able to use the `Address` type in `src/constants/types.ts`

Reply:
Nice one, done 👍